model/setpoint.go

```diff
@@ -64,14 +64,14 @@ type SetpointConstraintsListDataSelectorsType struct {
 }
 
 type SetpointDescriptionDataType struct {
-	SetpointId    *SetpointIdType   `json:"setpointId,omitempty" eebus:"key"`
-	MeasurementId *SetpointIdType   `json:"measurementId,omitempty"`
-	TimeTableId   *SetpointIdType   `json:"timeTableId,omitempty"`
-	SetpointType  *SetpointTypeType `json:"setpointType,omitempty"`
-	Unit          *ScaledNumberType `json:"unit,omitempty"`
-	ScopeType     *ScaledNumberType `json:"scopeType,omitempty"`
-	Label         *LabelType        `json:"label,omitempty"`
-	Description   *DescriptionType  `json:"description,omitempty"`
+	SetpointId    *SetpointIdType        `json:"setpointId,omitempty" eebus:"key"`
+	MeasurementId *SetpointIdType        `json:"measurementId,omitempty"`
+	TimeTableId   *SetpointIdType        `json:"timeTableId,omitempty"`
```

Comment:
`MeasurementId` and `TimeTableId` are documented as being foreign identifiers. Therefor the also need to get the tag annotation `eebus:"key"`, like `SetpointId` has

Reply:
Fixed. Thanks for noticing!
I added `eebus:"key"` to both foreign identifiers, `MeasurementId` and `TimeTableId`, in the `SetpointDescriptionDataType` class, just like it is for `SetpointId`.